types/artifacts.d.ts

```diff
@@ -241,6 +241,7 @@ declare global {
         failingTextLength: number;
         visitedTextLength: number;
         analyzedFailingTextLength: number;
+        /** Elements with a text node that failed size criteria. */
```

Comment:
I was unsure if these nodes were text nodes or not, so after confirming they weren't I added this comment to clarify.

Reply:
> after confirming they weren't

We might want to tweak the comment then, right now it reads like it's calling out that they're text nodes. I'm thinking maybe some emphasis on the relationship? `Elements that contain a text node`

Reply:
hmmm I'm curious how we interpret the comment differently. `with a` sounds the same as `that contain a` to me.

Reply:
I suppose it might be up to the emphasis in my head, I breeze by "with" so the relationship is lost by the time I'm reading while `that contain a` forces the connection to be made for me 🤷‍♂

Reply:
ok done